src/main/resources/cloudformation/vpc.yaml

```diff
@@ -71,6 +79,13 @@ Parameters:
     MaxLength: '20'
     MinLength: '9'
     Type: String
+  subnetCidrBlockForAz4:
```

Comment:
Should be conditional

Reply:
what would change though? this parameter won't be used if the az4 parameter isn't provided

Reply:
You're right